app/products/calls/utils.ts

```diff
@@ -201,3 +203,28 @@ export function isCallsStartedMessage(payload?: NotificationData) {
     // calls will be >= 0.21.0, and push proxy will be >= 5.27.0
     return (payload?.message === 'You\'ve been invited to a call' || callsMessageRegex.test(payload?.message || ''));
 }
+
+export const hasCaptions = (postProps?: Record<string, any> & { captions?: Caption[] }): boolean => {
+    return !(!postProps || !postProps.captions?.[0]);
+};
+
+export const getTranscriptionUri = (serverUrl: string, postProps?: Record<string, any> & { captions?: Caption[] }): {
+    track?: SubtitleTrack;
+    selected?: SelectedSubtitleTrack;
+} => {
+    // Note: We're not using hasCaptions above because this tells typescript that the caption exists later.
+    // We could use some fancy typescript to do the same, but it's not worth the complexity.
+    if (!postProps || !postProps.captions?.[0]) {
+        return {};
+    }
+
+    return {
+        track: {
+            title: postProps.captions[0].title,
+            language: postProps.captions[0].language,
+            type: TextTrackType.VTT,
+            uri: buildFileUrl(serverUrl, postProps.captions[0].file_id),
+        },
+        selected: {type: 'index', value: 0},
+    };
```

Comment:
How hard would it be to return all the available transcriptions vs assuming there's always going to be one?

Reply:
Not hard, I can do that. We'll only support one for now until we get UX designs for multiple (which matches webapp, I believe, right?)

Reply:
Yeah, just matching the web client is sufficient, thanks.